class.jetpack-cli.php

```diff
@@ -697,8 +703,57 @@ public function sync( $args, $assoc_args ) {
 				}
 
 				WP_CLI\Utils\format_items( 'table', $collection, array( 'option', 'value' ) );
+				break;
+			case 'settings':
+				WP_CLI::log( __( 'Sync Settings', 'jetpack' ) );
+				foreach( Jetpack_Sync_Settings::get_settings() as $setting => $item ) {
+					$settings[]  = array(
+						'setting' => $setting,
+						'value' => is_scalar( $item ) ? $item : json_encode( $item )
+					);
+				}
+
+				WP_CLI\Utils\format_items( 'table', $settings, array( 'setting', 'value' ) );
+
+			case 'disable':
+				Jetpack_Sync_Settings::update_settings( array( 'disable' => 1 ) );
+				WP_CLI::log( sprintf( __( 'Sync Disabled on %s', 'jetpack' ), get_site_url() ) );
+				break;
+			case 'enable':
+				// Don't set it via the Jetpack_Sync_Settings since that also
+				update_option( 'jetpack_sync_settings_disable', 0 );
+				WP_CLI::log( sprintf( __( 'Sync Enabled on %s', 'jetpack' ), get_site_url() ) );
+				break;
+			case 'reset':
+				Jetpack_Sync_Settings::update_settings( array( 'disable' => 1 ) );
```

Comment:
We disable sync, but I don't see it restarted. Should we output at the end a "use `wp jetpack sync start` to begin syncing again hint?